### PR TITLE
SAP SSO Release Notes PR

### PR DESCRIPTION
**Pull request by MarkvanMents**
For review by DES. Then needs a review by Adam to agree style.

### CODE REVIEW

File: content/releasenotes/developer-portal/index.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ title: "Developer Portal"
 
 ## 2018
 
+### July 3rd, 2018
+
+#### Improvements (SAP - Cloud Deployment Portal)
```
Comment (Adam-Dupaski) [inline content/releasenotes/developer-portal/index.md]: I don't think "SAP - Cloud Deployment Portal" is necessary to identify in heading. It's integrated into the Dev Portal, so we should just identify it as a general improvement.

File: content/releasenotes/developer-portal/index.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ title: "Developer Portal"
...
+
+* To improve integration between Mendix and SAP we now enforce using the same username (email address) for Mendix and SAP. You will need to use the same username the next time you need to provide your credentials. 
+  *  This is currently implemented only for SAP regions **eu10** and **us10**.
```
Comment (Adam-Dupaski) [inline content/releasenotes/developer-portal/index.md]: Second bullet probably not necessary, just put the last sentence in the main bullet/paragraph.

File: content/releasenotes/developer-portal/index.md (hunk 1 above)
Comment (MarkvanMents) [reply content/releasenotes/developer-portal/index.md]: It only affects SAP users. but I'll make sure that is clear in the text. Thanks for the comments.

Comment (Adam-Dupaski) [reply content/releasenotes/developer-portal/index.md]: Sounds good. Anything like that should be made clear in the text, that way we can keep the headings consistent and generic.

**Comment by MarkvanMents** [comment]
Changes from Adam and review with Mehran.
Changes referred to in release notes have been released.
Releasing Document.

**Pull request merged**